### DISCUSSION TITLE
start without GPU

### DISCUSSION THREAD

**Discussion by haveitallism** [cyberbotics/webots#6175] [Q&A]
hello, I want to start webots on a linux machine without a GPU, is this possible?

**Comment by omichel** [comment]
Yes, this is possible, but the rendering will be performed by the mesa library and it will be slow, especially if you are simulating cameras or lidars.

**Comment by haveitallism** [comment]
If possible, how should I do, I don’t need to simulate cameras or lidars, I want to use ros2 launch file start webots, how should I configure it?

**Reply by omichel** [reply]
There is nothing special to do, it should work out-of-the-box.
